en/docs/manage-databases-and-caches/choreo-managed-postgresql-databases.md

```diff
@@ -1,21 +1,23 @@
-# Choreo-Managed PostgreSQL Databases
+# Choreo-Managed PostgreSQL Databases/Vector Databases
 
-PostgreSQL on Choreo offers fully Choreo-managed, efficient object-relational databases on AWS, Azure, GCP, and Digital Ocean.
+PostgreSQL on Choreo offers fully Choreo-managed, efficient object-relational databases on AWS, Azure, GCP, and Digital Ocean which also can be used as a Vector database. 
```

Comment:
How about we go with something along the lines of:
PostgreSQL on Choreo offers fully Choreo-managed, efficient object-relational databases on AWS, Azure, GCP, and Digital Ocean. Additionally, Choreo allows you to create fully-managed PostgreSQL vector databases if you want to perform efficient vector similarity search.